### PR TITLE
[Master]Add time stamp in image query

### PR DESCRIPTION
**Pull request by SharpRazor**

### CODE REVIEW

File: zvmsdk/returncode.py [hunk 1]
```diff
@@ -217,6 +217,8 @@
                21: "Image Export error: Failed to copy image file to remote "
                    "host with reason: %(msg)s",
                22: "Export image to local file system failed: %(err)s",
+               23: "Image file of %(img)s does not exist, "
```
Comment (qianxi2017) [inline zvmsdk/returncode.py]: test

Comment (SharpRazor) [reply zvmsdk/returncode.py]: ok, I can see you.

**Comment by bjhuangr** [comment]
@jichenjc Please note this change will change image_query outputs, potential in-compatible problem.

**Comment by SharpRazor** [comment]
> @jichenjc Please note this change will change image_query outputs, potential in-compatible problem.

I updated document, so let's do full test when FP1 be built.

**Comment by jichenjc** [comment]
@bjhuangr I don't know the detail of this PR , need check

@SharpRazor we have no other way to do because we have to query the info, correct? 
And this will only add additional field into output, nothing changed, correct?

**Comment by SharpRazor** [comment]
@jichenjc @bjhuangr yes, we must know the timestamp from zcc. So I have updated the document to add the description of timestamp.

**Comment by qianxi2017** [comment]
> @bjhuangr I don't know the detail of this PR , need check
> 
> @SharpRazor we have no other way to do because we have to query the info, correct?
> And this will only add additional field into output, nothing changed, correct?

And this fix imports a new potential exception while doing the image query, which might be in-compatible. So, @SharpRazor I think we should mute this exception in Line: https://github.com/openmainframeproject/python-zvm-sdk/pull/100/files#diff-00999ea921397098b8dc799ec59174cfR2268

**Comment by SharpRazor** [comment]
@qianxi2017 you are right.

**Comment by SeanHQF** [comment]
lgtm

**Pull request merged**